primitive-types/impls/serde/benches/input.rs

```diff
@@ -0,0 +1,18 @@
+
+/// Hexadecimal string 65536 chars (32768 bytes)
```

Comment:
this is actually 32k chars (16k bytes)
(I was surprised to see HEX_65536_CHARS test being 2x slower than HEX_16384_CHARS)